<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 14 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{mso-style-link:"Comment Text Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	font-size:10.0pt;
	font-family:"Calibri","sans-serif";}
a:link, span.MsoHyperlink
	{color:blue;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:purple;
	text-decoration:underline;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{mso-style-link:"Comment Subject Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	font-size:10.0pt;
	font-family:"Calibri","sans-serif";
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-link:"Balloon Text Char";
	margin:0in;
	margin-bottom:.0001pt;
	font-size:8.0pt;
	font-family:"Tahoma","sans-serif";}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-link:"Balloon Text";
	font-family:"Tahoma","sans-serif";}
span.CommentTextChar
	{mso-style-name:"Comment Text Char";
	mso-style-link:"Comment Text";}
span.CommentSubjectChar
	{mso-style-name:"Comment Subject Char";
	mso-style-link:"Comment Subject";
	font-weight:bold;}
.MsoChpDefault
	{font-family:"Calibri","sans-serif";}
.MsoPapDefault
	{margin-bottom:10.0pt;
	line-height:115%;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.WordSection1
	{page:WordSection1;}
-->
</style>

</head>

<body lang=EN-US link=blue vlink=purple>

<div class=WordSection1>
<center>
<p class=MsoNormal><img width=598 height=166 id="Picture 2"
src="welcome_image001.jpg"></p>
</center>
<p class=MsoNormalCxSpMiddle style='margin-top:12.0pt;text-align:justify;
line-height:normal;background:#EEECE1'><span style='font-family:"Arial","sans-serif"'>Welcome
to the Antigen receptor Galaxy (ARGalaxy) </span><span lang=EN
style='font-family:"Arial","sans-serif";color:#2A2A2A'>tool to </span><span
style='font-family:"Arial","sans-serif"'>automate</span><span lang=EN
style='font-family:"Arial","sans-serif";color:#2A2A2A'> the analysis and visual
reporting of T- and B-cell receptor repertoire sequencing data.  ARGalaxy is a
collaboration between the Departments of Immunology and Bioinformatics at
Erasmus MC. If you have used ARGalaxy in your publication then IJspeert and van Schouwenburg et al, 2017 [PMID: 28416602] and Moorehouse et al, 2014 [PMID: 25495099]</span></p>

<p class=MsoNormalCxSpMiddle style='margin-top:12.0pt;text-align:justify;
line-height:normal;background:#EEECE1'><span lang=EN style='font-family:"Arial","sans-serif";
color:#2A2A2A'>ARGalaxy is maintained by David van Zessen: </span><a
href="mailto:d.vanzessen@erasmusmc.nl"><span lang=EN style='font-family:"Arial","sans-serif"'>d.vanzessen@erasmusmc.nl</span></a><span
lang=EN style='font-family:"Arial","sans-serif";color:#2A2A2A'> and the
Bioinformatics Department at the Erasmus MC</span></p>

<br />
<br />
<br />
<br />
<br />

<div style='border:none;border-bottom:solid windowtext 1.0pt;padding:0in 0in 1.0pt 0in'>

<p class=MsoNormalCxSpMiddle style='margin-top:12.0pt;text-align:justify;
line-height:normal;border:none;padding:0in'><span lang=EN style='font-family:
"Arial","sans-serif";color:#2A2A2A'> </span></p>

</div>

<p class=MsoNormalCxSpMiddle align=center style='margin-top:12.0pt;text-align:
center;line-height:normal'><b><span style='font-size:16.0pt;font-family:"Arial","sans-serif";
color:#548DD4'>Using ARGalaxy</span></b></p>

<p class=MsoNormalCxSpMiddle style='margin-top:12.0pt;text-align:justify;
line-height:normal'><span style='font-family:"Arial","sans-serif"'>Antigen
receptor Galaxy (ARGalaxy) is a </span><span lang=EN style='font-family:"Arial","sans-serif";
color:#2A2A2A'>tool to </span><span style='font-family:"Arial","sans-serif"'>automate</span><span
lang=EN style='font-family:"Arial","sans-serif";color:#2A2A2A'> the analysis
and visual reporting of T- and B-cell receptor repertoire sequencing data.</span></p>

<p class=MsoNormalCxSpMiddle style='margin-top:12.0pt;text-align:justify;
line-height:normal'><span style='font-family:"Arial","sans-serif";color:black'>ARGalaxy
consists of four parts: the demultiplex tool, the IMGT concatenate tool, the </span><span
style='font-family:"Arial","sans-serif"'>immune repertoire pipeline and the
SHM&amp;CSR pipeline. T<span style='color:black'>ogether they allow the
analysis of all different aspects of the immune repertoire. All pipelines can
be run independently or combined, dependent on the available data and the
question of interest</span></span></p>

<center>
<img border=0 width=1008 height=783 id="Picture 1" src="welcome_image002.png">
</center>
<p class=MsoNormalCxSpMiddle style='margin-top:12.0pt;text-align:justify;
line-height:normal'><b><span style='font-family:"Arial","sans-serif";
color:#1F497D'>Demultiplex tool</span></b></p>

<p class=MsoNormal style='text-align:justify'><span style='font-family:"Arial","sans-serif";
color:black'>The demultiplex tool demultiplexes .sff files with the option of
sequence trimming with both downloadable FASTA and FASTQ files available as
output. This tool is based on the freely available sff2fastq (</span><a
href="https://github.com/indraniel/sff2fastq"><span style='font-family:"Arial","sans-serif"'>https://github.com/indraniel/sff2fastq</span></a><span
style='font-family:"Arial","sans-serif";color:black'>), and FASTX-Toolkit
software (</span><span class=MsoHyperlink><span style='font-family:"Arial","sans-serif"'>http://hannonlab.cshl.edu­
/fastxtoolkit/commandline.html</span></span><span style='font-family:"Arial","sans-serif";
color:black'>), and includes the FastQC tool to check the quality of your data
(</span><a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/"><span
style='font-family:"Arial","sans-serif"'>http://www.bioinformatics.­babraham.ac.uk/projects/fastqc/</span></a><span
style='font-family:"Arial","sans-serif";color:black'>).</span></p>

<p class=MsoNormalCxSpMiddle style='text-align:justify;line-height:normal'><b><span
style='font-family:"Arial","sans-serif";color:#1F497D'>IMGT concatenate tool</span></b></p>

<p class=MsoNormalCxSpMiddle style='text-align:justify;line-height:normal'><span
style='font-family:"Arial","sans-serif"'>The IMGT concatenate tool can be used
to combine multiple .txz files (as downloaded from IMGT/High V-Quest </span><a
href="http://www.imgt.org/HighV-QUEST"><span style='font-family:"Arial","sans-serif"'>http://www.imgt.org/HighV-QUEST</span></a><span
style='font-family:"Arial","sans-serif"'> ) into a single .txz file.</span></p>

<p class=MsoNormalCxSpMiddle style='text-align:justify;line-height:normal'><b><span
style='font-family:"Arial","sans-serif";color:#1F497D'>Immune repertoire
pipeline</span></b></p>

<p class=MsoNormalCxSpMiddle style='text-align:justify;line-height:normal'><span
style='font-family:"Arial","sans-serif"'>The immune repertoire pipeline is an
extended version of our previously published IGGalaxy virtual machine
(Moorhouse et al, 2014) which was developed to visualize V(D)J gene usage. It
now allows analysis of not only B-cell, but also T-cell receptor rearrangements
and allows the visualization of CDR3 characteristics (length and amino acid
usage), junction characteristics and the calculation of the diversity of the
immune repertoire (Boyd et al, 2009). </span><span lang=EN style='font-family:
"Arial","sans-serif"'>The Immune Repertoire pipeline </span><span
style='font-family:"Arial","sans-serif"'>requires </span><span lang=EN
style='font-family:"Arial","sans-serif"'>fasta files, </span><span
style='font-family:"Arial","sans-serif"'>IMGT/HighV-QUEST output files, or
output files from the SHM&amp;CSR pipeline as input files and gives results as
a set of  tables and graphs accessible in html format.</span></p>

<p class=MsoNormalCxSpMiddle style='text-align:justify;line-height:normal'><b><span
style='font-family:"Arial","sans-serif";color:#1F497D'>SHM&amp;CSR pipeline</span></b></p>

<p class=MsoNormalCxSpMiddle style='text-align:justify;line-height:normal'><span
style='font-family:"Arial","sans-serif"'>The SHM&amp;CSR pipeline can be used for
the analysis of somatic hypermutations (SHM) and/or class switch recombination
(CSR) in B-cell receptor rearrangements. This SHM&amp;CSR pipeline provides the
analysis of SHM levels and patterns, antigen selection (including BASELINe
(Yaari et al, 2012)), clonality (using Change-O (Gupta et al, 2015)) and CSR.
It requires IMGT/HighV-QUEST .txz files as  input files, parses the relevant
information from these files, and provides a html output as a set of tables and
graphs. </span></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><b><span
style='font-family:"Arial","sans-serif";color:#1F497D'>Test data sets</span></b></p>

<p class=MsoNormalCxSpMiddle style='text-align:justify;line-height:normal'><span
style='font-family:"Arial","sans-serif"'>The test data set can be imported from the 'Data libraries' under 'Shared data' or by using this link: <a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/library/list'>https://bioinf-galaxian.erasmusmc.nl/argalaxy/library/list</a> <br />
Histories in which the test data sets are used in the demultiplex tool, the IMGT concatenate tool, the immune repertoire pipeline or the SHM&CSR pipeline can be imported from the 'Histories' under 'Shared data' or by using this link: <a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/u/davidvanzessen/h/test-data-sets'>https://bioinf-galaxian.erasmusmc.nl/argalaxy/u/davidvanzessen/h/test-data-sets</a></span></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><b><span
style='font-family:"Arial","sans-serif";color:#1F497D'>Tutorials: </span></b></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><span style='font-family:
"Arial","sans-serif";color:#2A2A2A'><a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/datasets/e144222fed44799a/display/?preview=True'>Demultiplex tool</a></span></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><span style='font-family:
"Arial","sans-serif";color:#2A2A2A'><a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/datasets/915ae9a80309f157/display/?preview=True'>Concatenate Tool</a></span></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><span style='font-family:
"Arial","sans-serif";color:#2A2A2A'><a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/datasets/13cea0e6d733b865/display/?preview=True'>Immune repertoire pipeline</a></span></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><span style='font-family:
"Arial","sans-serif";color:#2A2A2A'><a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/datasets/fd767e8b7355e21a/display/?preview=True'>SHM &amp; CSR pipeline</a></span></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><b><span
style='font-family:"Arial","sans-serif";color:#1F497D'>Detailed description of
the tools and pipelines:</span></b></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><span style='font-family:
"Arial","sans-serif";color:#2A2A2A'><a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/u/davidvanzessen/p/demultiplex-page'>Demultiplex tool</a></span></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><span style='font-family:
"Arial","sans-serif";color:#2A2A2A'><a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/u/davidvanzessen/p/imgt-concatenate-page'>Concatenate Tool</a></span></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><span style='font-family:
"Arial","sans-serif";color:#2A2A2A'><a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/u/davidvanzessen/p/immune-repertoire-page'>Immune repertoire pipeline</a></span></p>

<p class=MsoNormalCxSpMiddle style='line-height:normal'><span style='font-family:
"Arial","sans-serif";color:#2A2A2A'><a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/u/davidvanzessen/p/shmcsr-page'>SHM &amp; CSR pipeline</a></span></p>

</div>

</body>

</html>
